docs/sources/mimir/configure/configure-otel-collector.md

```diff
@@ -107,6 +107,41 @@ service:
       exporters: [..., prometheusremotewrite]
 ```
 
+## Work with default OpenTelemetry labels
+
+OpenTelemetry metrics use resource attributes to describe the set of characteristics associated with a given resource, or entity producing telemetry data. For example, a host resource might have multiple attributes, including an ID, an image, and a type.
+
+To optimize the storage of and ability to query this data, Mimir automatically assigns the following OTel resource attributes as labels at the time of ingestion, with periods (`.`) replaced by underscores (`_`):
```

Comment:
Mimir has to be configured to do this, whereas in Grafana Cloud we have a default list of promoted resource attributes:
```suggestion
To optimize the storage of and ability to query this data, Mimir can be configured to automatically promote OTel resource attributes to labels at the time of ingestion, with periods (`.`) replaced by underscores (`_`). In Grafana Cloud, the following resource attributes are promoted:
```

Reply:
Great call out. I changed the wording a little, but went with the general idea here.